generators/app/templates/src/app/components/CustomText/styles.js

```diff
@@ -0,0 +1,55 @@
+import { StyleSheet } from 'react-native';
+
+import { blue, white, green, gray, black, transparent } from '../../../utils/colors';
+import { IS_SMALL_DEVICE } from '../../../utils/constants';
+
+const amountToResize = 2 * IS_SMALL_DEVICE;
+
+// Custom Text Style
+// You can add styles to Base like Family Font to be the Text styles base!
+// If you wanna a particular style you can add this like colors or sizes and then you have to
```

Comment:
`if you want to add a custom style, you need to add it here and in VARIANTS Array in index.js` ?